### ISSUE TITLE
minify:clear returns exception error

### ISSUE THREAD

**Issue by vitor-ao** [doublethreedigital/minify#10]
When running the command `please minify:clear`, it returns the following error:
```shell
[prompt]$ php please minify:clear
Clearing your Minify cache..

   BadMethodCallException

  Method DoubleThreeDigital\Minify\Console\Commands\ClearMinifyCache::getFilename does not exist.

  at vendor/laravel/framework/src/Illuminate/Macroable/Traits/Macroable.php:103
     99▕      */
    100▕     public function __call($method, $parameters)
    101▕     {
    102▕         if (! static::hasMacro($method)) {
  ➜ 103▕             throw new BadMethodCallException(sprintf(
    104▕                 'Method %s::%s does not exist.', static::class, $method
    105▕             ));
    106▕         }
    107▕

  • Bad Method Call: Did you mean DoubleThreeDigital\Minify\Console\Commands\ClearMinifyCache::getName() ?

      +17 vendor frames
  18  please:37
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
```

Happens on my local environment (Windows) and staging (Linux).

## Steps to reproduce

1. Run `php please minify:clear` on the Statamic directory.

## Expected behaviour

I expected to clear the minified files.

## Environment information

**Statamic version:** 3.0.19 Pro
_run `php please -V` to find it._

**PHP version:** 7.4.11
_run `php -v` to find it_

**Operating System:**
- [ ] MacOS
- [x] Linux
- [x] Windows

**Browser:**
- [ ] Chrome
- [x] Firefox
- [ ] Safari

**Comment by damcclean** [comment]
Thanks for reporting! I'll look into it this week.

**Comment by damcclean** [comment]
Fixed in [v3.2.2](https://github.com/doublethreedigital/minify/releases/tag/v3.2.2)
